src/main/java/com/puppycrawl/tools/checkstyle/checks/regexp/RegexpMultilineCheck.java

```diff
@@ -73,33 +73,76 @@
  * </li>
  * </ul>
  * <p>
+ * To run the check with its default configuration (no matches will be):
+ * </p>
+ * <pre>
+ * &lt;module name=&quot;RegexpMultiline&quot;/&gt;
+ * </pre>
+ * <p>Example: </p>
+ * <pre>
+ * void method() {
+ *   int i = 5; // OK
+ *   System.out.println(i); // OK
+ * }
+ * </pre>
+ * <p>
  * To configure the check to find calls to print to the console:
  * </p>
  * <pre>
  * &lt;module name="RegexpMultiline"&gt;
- *   &lt;property name="format"
- *     value="System\.(out)|(err)\.print(ln)?\("/&gt;
+ *   &lt;property name="format" value="System\.(out)|(err)\.print(ln)?\("/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * void method() {
+ *   System.out.print("Example"); // violation
+ *   System.err.println("Example"); // violation
+ *   System.out.print
+ *     ("Example"); // violation
+ *   System.err.println
+ *     ("Example"); // OK
+ *   System
+ *   .out.print("Example"); // OK
+ *   System
+ *   .err.println("Example"); // violation
+ *   System.
+ *   out.print("Example"); // OK
+ *   System.
+ *   err.println("Example"); // violation
+ * }
+ * </pre>
+ * <p>
  * To configure the check to match text that spans multiple lines,
  * like normal code in a Java file:
  * </p>
  * <pre>
  * &lt;module name="RegexpMultiline"&gt;
  *   &lt;property name="matchAcrossLines" value="true"/&gt;
- *   &lt;property name="format" value="System\.out.*print\("/&gt;
+ *   &lt;property name="format" value="System\.out.*?print\("/&gt;
  * &lt;/module&gt;
  * </pre>
  * <p>
- * Example of violation from the above config:
+ * Example:
  * </p>
  * <pre>
  * void method() {
- *   System.out. // violation
- *   print("Example");
- *   System.out.
- *   print("Example");
+ *   System.out.print("Example"); // violation
+ *   System.err.println("Example");
+ *   System.out.print // violation
```

Comment:
please do me favor and vertically align trailing comment, not not mix with code visually.

this case force me to recheck on local execution of this example, as I did not noticed this comment.